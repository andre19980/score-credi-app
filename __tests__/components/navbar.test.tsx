import "@testing-library/jest-dom";
import { render } from "@testing-library/react";
import Navbar from "@/components/navbar";

describe("Navbar", () => {
  it("renders component", () => {
    const { container } = render(<Navbar />);
    expect(container).toMatchSnapshot();
  });
});
