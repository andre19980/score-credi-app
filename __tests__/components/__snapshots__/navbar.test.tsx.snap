// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar renders component 1`] = `
<div>
  <nav
    aria-label="Navbar"
    class="bg-gray-50 dark:bg-gray-900 fixed w-full z-20 top-0 start-0 h-20 flex"
    role="navigation"
  >
    <div
      class="max-w-screen-xl flex flex-wrap items-center p-4"
    >
      <img
        alt="Blipay logo"
        data-nimg="1"
        decoding="async"
        fetchpriority="high"
        height="42"
        src="/_next/image?url=%2Fimg.jpg&w=384&q=75"
        srcset="/_next/image?url=%2Fimg.jpg&w=256&q=75 1x, /_next/image?url=%2Fimg.jpg&w=384&q=75 2x"
        style="color: transparent;"
        width="140"
      />
    </div>
  </nav>
</div>
`;
